runelite-client/src/main/java/net/runelite/client/plugins/driftnet/DriftNetPlugin.java

```diff
@@ -175,12 +198,6 @@ private void updateDriftNetVarbits()
 			net.setStatus(status);
 			net.setCount(count);
 		}
-
-		// When you collect any loot, all tags become invalidated
-		if (client.getVar(Varbits.DRIFT_NET_COLLECT) != 0)
-		{
-			taggedFish.clear();
-		}
```

Comment:
Are you sure this wrong? It seemed correct when i tested it, though it's possible i was mistaken.

Reply:
It's not wrong, it's just that the fish shoal stay tagged even when you collect the loot. The fish shoal only become untagged when they are adjacent to a drift net that is not actively catching (= full, unset) or when they have been tagged for a certain amount of ticks, which I was unable to determine but would seem to be somewhere randomly between 40 and 100.

Reply:
Alright, how did you time the ticks?

Reply:
I added an overlay to the fish shoal which showed the tick it was tagged (value of taggedFish).

Reply:
I agree with all the other comments, do I add these changes in a new commit(s) or do I amend the current commits?

Reply:
I get that, but i was more interested in how you determined when a shoal was no longer tagged? Was it when it moved away from the net, or just didn't move towards it for example?

Reply:
I'd prefer it if you amend your current commits :)

Reply:
Visually when a shoal was 2 or less tiles away from a drift net but did not go in it. Which means it was probably untagged for a longer time, but just never moved close enough to a net. That's why it's hard to determine how many ticks exactly it takes for a shoal to become untagged. But in this implementation it seldomly happens that a tagged shoal is actually not tagged.

Maybe if you have some time to test this implementation you can determine what causes a shoal which seems to be tagged to not actually be. It does not happen very often so I was unable to determine this case. It could be just a tick timeout or something else.

Reply:
No i trust you, i was mostly interested in how you did it because i had the same problem, it's very hard to determine. Sounds good.